src/src/rx-serial/SerialMavlink.cpp

```diff
@@ -42,6 +42,7 @@ void SerialMavlink::sendQueuedData(uint32_t maxBytesToSend)
 
 #else // ESP-based targets
 
+#define MAVLINK_COMM_NUM_BUFFERS 1
 #include "common/mavlink.h"
```

Comment:
Why not just `#include "MAVlink.h"` and have MAVLINK_COMM_NUM_BUFFERS defined in a single place? Also no need to `#include "common/mavlink.h"` then.

```C++
...
#else // ESP-based targets

#include "MAVLink.h"

#define MAV_FTP_OPCODE_OPENFILERO 4
...
```

Reply:
@mha1 Very shortly we will need different values on TX and RX. https://github.com/ExpressLRS/ExpressLRS/pull/2841, https://github.com/ExpressLRS/ExpressLRS/pull/2822